src/domain/flow-system/services/src/messages/flow_message_producers.rs

```diff
@@ -0,0 +1,17 @@
+// Copyright Kamu Data, Inc. and contributors. All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+pub const MESSAGE_PRODUCER_KAMU_FLOW_CONFIGURATION_SERVICE: &str =
```

Comment:
Nitpic: These long constants names make my head heart. Perhaps instead of giving them unique names we could utilizes the module namespaces?

e.g. 
```rust
kamu_flow_system::domain::services::flow::MESSAGE_PRODUCER_ID
kamu_flow_system::domain::services::flow_configuration::MESSAGE_PRODUCER_ID
```

Reply:
We are going to have "glob reexport" issues this way, besides, I do not see value in this change, It's looking roughly identical.